packages/bull-vault/src/BullStrategy.sol

```diff
@@ -137,7 +137,7 @@ contract BullStrategy is ERC20, LeverageBull {
     function setShutdownContract(address _shutdownContract) external onlyOwner {
         require(_shutdownContract != address(0), "BS6");
 
-        emit SetShutdownContract(shutdownContract, _shutdownContract);
+        emit SetShutdownContract(_shutdownContract, shutdownContract);
```

Comment:
this is now backwards _shutdownContract is the new one, and shutdownContract is the old one